Capri.Web/src/components/navStudentItems.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<div id="studentBar">
+		<v-list-item
+			v-for="(filter, i) in filters"
+			:key="i"
+			class="paddingAndMarginZero px-3"
+		>
+			<v-list-item-action
+				color="blue"
+				class="paddingAndMarginZero my-4"
+                style="width: 310px;"
+			>
+				<v-select
+					:items="filter.values"
+					:label="filter.name"
+					:model="filter.chosen"
+					v-on:change="changeRoute(i)"
+					color="blue"
+					class="paddingAndMarginZero"
+				>
+				</v-select>
+			</v-list-item-action>
+		</v-list-item>
+	</div>
+</template>
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+@Component
+export default class NavStudentItems extends Vue {
+    public filters = [
+        {
+            name: 'Faculty',
+            values: ['Computer Science'],
+            chosen: false,
+        },
+        {
+            name: 'Field of study',
+            values: ['IT', 'Robotics', 'Bioinformatics'],
+            chosen: false,
+        },
+        {
+            name: 'Degree',
+            values: ['Bacheloer', 'Master'],
+            chosen: false,
+        },
+        {
+            name: 'Type',
+            values: ['Full time', 'Part time'],
+            chosen: false,
+        },
+    ];
+    public data() {
+        return {
+            isAllSelected: 'nie',
+        };
+    }
+    public changeRoute(which): void {
```

Comment:
`which` what's type of that? Another thing i think it's a very bad name. It should be something like `rowIndex`

Reply:
Also can you explain what actually that method do?

Reply:
It was method to change route when all of the filtering options was selected. Now it's not important – removed.